src/commonMain/kotlin/exchange.dydx.abacus/state/manager/configs/V4StateManagerConfigs.kt

```diff
@@ -55,7 +55,8 @@ class V4StateManagerConfigs(
                          "status":"/v1/status"
                       },
                       "configs":{
-                         "markets":"/configs/markets.json"
+                         "markets":"/configs/markets.json",
+                         "assets": "https://66iv2m87ol.execute-api.ap-northeast-1.amazonaws.com/mainnet/metadata-service/v1/info"
```

Comment:
Do you think it's better to put this into env.json?  In case the host ever changes, we can just update the config json.  Assets are essential to the app.